src/cfnlint/data/CloudSpecs/us-east-1.json

```diff
@@ -13888,7 +14123,6 @@
       "Properties": {
         "Action": {
           "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-databrew-recipe-recipestep.html#cfn-databrew-recipe-recipestep-action",
-          "PrimitiveType": "Json",
```

Comment:
might need to patch [`AWS::DataBrew::Recipe`](https://code.amazon.com/packages/AWSCloudFormationResourceProvidersElixir/blobs/mainline/--/aws-databrew-recipe/aws-databrew-recipe.json)

Reply:
[can temporarily disable Github actions](https://github.com/aws-cloudformation/cfn-python-lint/actions?query=workflow%3A.github%2Fworkflows%2Fmaintenance.yaml) if we need to patch [Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) bugs any week

Reply:
I'm not seeing the issue here.  What am I missing?

Reply:
[`PrimitiveType` dropped](https://github.com/awslabs/goformation/issues/300#issuecomment-715467006)

Reply:
@PatMyron talked and I think we are good here. Resolving